mitodl/bootcamp-ecommerce
# Quiet template absent variable errors - #974

#### What are the relevant tickets?
Fixes #974 

#### What's this PR do?
Fixes a quiet error in template rendering when accessing a variable that wasn't defined in the first place.

#### How should this be manually tested?
Visit any page with debugging/logging level set to DEBUG/INFO and verify no errors pertaining to `CSOURCE_PAYLOAD` missing in template context are raised.
